drake/doc/ubuntu.rst

```diff
@@ -10,10 +10,6 @@ Install the prerequisites::
     sudo add-apt-repository -y ppa:ubuntu-toolchain-r/test
     wget -q -O - http://llvm.org/apt/llvm-snapshot.gpg.key | sudo apt-key add -
 
-    # Ubuntu 12.04 LTS (Precise)
-    sudo add-apt-repository -y "deb http://llvm.org/apt/precise/ llvm-toolchain-precise-3.7 main"
-    sudo add-apt-repository -y ppa:kalakris/cmake
-
     # Ubuntu 14.04 LTS (Trusty)
```

Comment:
This comment still appears here and on line 26.  I am unclear on what its purpose is.